src/components/Modals/RulesModal.vue

```diff
@@ -116,25 +116,130 @@
+                      <div>
+                        <img src="../../../static/cardImg/PowerOutage.png" alt="Instruction 1 Card" class="cardImage">
+                      </div>
+
+                      <div>
+                        <ol>
+                          <li>Power Outage cards let you stop another player from playing instruction cards.</li>
+                          <li>When you play a Power Outage card, you choose which player will be impacted. </li>
+                          <li>You cannot give a power outage to someone with a generator card active.</li>
+                        </ol>
+                        <b>Relation to computer programming:</b>
+                        Sometimes when the weather is bad you can have a power outage. This makes it so that you can't create programs.
+                        A backup battery or a generator can resolve this and allow you to continue programming, which is why the battery
+                        backup and generator cards reverse this program.
+                      </div>
+                    </li>
+                    <li><h5>Battery Backup</h5>
+                      <div>
+                        <img src="../../../static/cardImg/BatteryBackup.png" alt="Instruction 1 Card" class="cardImage">
+                      </div>
+
+                      <div>
+                        <ol>
+                          <li>Battery Backup cards reverse the effects of Power Outage cards.</li>
+                        </ol>
+                        <b>Relation to computer programming:</b>
+                        Sometimes when the weather is bad you can have a power outage. This makes it so that you can't create programs.
+                        A backup battery or a generator can resolve this and allow you to continue programming, which is why the battery
+                        backup and generator cards reverse this program.
                       </div>
                     </li>
+                    <li><h5>Overclocking</h5>
+                      <div>
+                        <img src="../../../static/cardImg/OverClock.png" alt="Instruction 1 Card" class="cardImage">
+                      </div>
+
+                      <div>
+                        <ol>
+                          <li>Overclocking cards allow you to double your current score.</li>
+                        </ol>
+                        <b>Relation to computer programming:</b>
+                        When you overclock your computer your computer runs faster, allowing your programs to run much quicker.
+                      </div>
+                    </li>
+
+
+                  <li><h5>Generator</h5>
+                    <div>
+                      <img src="../../../static/cardImg/Generator.png" alt="Instruction 1 Card" class="cardImage">
+                    </div>
+                    <div>
+                      <ol>
+                        <li>Generator cards protect you from all Power Outage cards.</li>
```

Comment:
Need to make it clear that a Battery Backup only works once a Power Outage has been  played and then it is discarded, but a Generator remains until the end of the game and prevents opponents from playing a Power Outage on them in the future. The same is true for the other two "safety" cards.

Reply:
Yeah, thats how I programmed it. Is this not clear in the rules?

Reply:
Are you saying that a generator shouldn't take off a current power outage?